CMakeLists.txt

```diff
@@ -56,7 +56,7 @@ endif()
 
 # SDK Clients and tests
 add_subdirectory(sdk/storage/blobs)
-add_subdirectory(sdk/iot/core)
+add_subdirectory(sdk/iot/common)
```

Comment:
Make sure to verify and fix up and links/urls in READMEs that might still be pointing to `.../iot/core...` I don't know if any exist, but it would be good to double check mentions of `core`.

Reply:
I did a find&replace throughout the whole iot SDK. Everything that have az_iot_core and iot/core got renamed.